src/styles/brackets.less

```diff
@@ -672,6 +673,10 @@ a, img {
     background-image: url("images/discuss-icon.svg");
```

Comment:
shouldn't these and related code be removed as we are deleting the discuss icon flow?